docs/core/setup/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS - .NET Core
+description: Learn where and what to install for .NET Core on Windows, Linux, and macOS versions. Discover the dependencies required to develop, deploy, and run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+---
+
+# Download and install .NET Core
+
+This article provides information on how to download and install .NET Core. There are two parts to .NET Core, the **runtime** which is used to run apps, and the **SDK** which is used to create apps. The SDK includes the runtime.
```

Comment:
```suggestion
This article provides information on how to download and install .NET Core. There are two parts to .NET Core: the **runtime**, which is used to run apps, and the **SDK**, which is used to create apps. The SDK includes the runtime.
```

Reply:
This could be in a list as well

Reply:
I think if there are two items, it's ok to have this as inline text. If there were three items, I think it should be converted.